pysph/examples/db_exp_data.py

```diff
@@ -7,12 +7,22 @@
 Plane", J. C. Martin and W. J. Moyce, Philososphical Transactions of the Royal
 Society of London Series A, 244, 312--324 (1952).
 
-and
-
 "Moving-Particle Semi-Implicit Method for Fragmentation of Incompressible
 fluid", S. Koshizuka and Y. Oka, Nuclear Science and Engineering, 123,
 421--434 (1996).
 
+"B.Buchner, Green Water on Ship - Type Off shore Structures, Ph.D.thesis,
```

Comment:
Please add the DOI's for each of these papers, and please do so for the ones above also. Thanks!

Reply:
Perhaps a note on how this was extracted so people are aware?